### PR TITLE
Unused import

### PR DESCRIPTION
**Pull request by qmonmert**
## Checklist
Before submitting this PR, please check the following points:
- [ ] I have added unit and integration tests for my change
- [ ] All unit and integration tests in the module I have added/changed are green
- [ ] All unit and integration tests in the [core](https://github.com/langchain4j/langchain4j/tree/main/langchain4j-core) and [main](https://github.com/langchain4j/langchain4j/tree/main/langchain4j) modules are green
- [ ] I have added/updated the [documentation](https://github.com/langchain4j/langchain4j/tree/main/docs/docs)
- [ ] I have added an example in the [examples repo](https://github.com/langchain4j/langchain4j-examples) (only for "big" features)
- [ ] I have added my new module in the [BOM](https://github.com/langchain4j/langchain4j/blob/main/langchain4j-bom/pom.xml) (only when a new module is added)

### CODE REVIEW

**Comment by geoand** [comment]
Any chance you can rebase this onto `main`?

Thanks

**Comment by qmonmert** [comment]
@geoand done

File: langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockLlamaChatModel.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package dev.langchain4j.model.bedrock;
 
 import dev.langchain4j.model.bedrock.internal.AbstractBedrockChatModel;
-import dev.langchain4j.model.bedrock.internal.BedrockChatModelResponse;
+
```
Comment (Wadimz) [inline langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockLlamaChatModel.java]: nit: shouldn't it be together with other imports without empty line as before?

**Review by Wadimz — APPROVED** [review]

File: langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockLlamaChatModel.java (hunk 1 above)
Comment (qmonmert) [reply langchain4j-bedrock/src/main/java/dev/langchain4j/model/bedrock/BedrockLlamaChatModel.java]: @Wadimz done

**Pull request closed without merging**